backend/src/web-server/controllers/oibus.controller.ts

```diff
@@ -57,41 +57,59 @@ export default class OibusController extends AbstractController {
   }
 
   async addValues(ctx: KoaContext<void, void>): Promise<void> {
```

Comment:
Here `addValues` and `addFile` are two separate methods called here:
https://github.com/OptimistikSAS/OIBus/blob/f1d62c74f1aeacb69f729cd7df823ba88530e2e0/backend/src/web-server/routes/index.ts#L88-L89

Maybe we make it into one `/api/add-external-content` endpoint, with a query param of `type`?
This way, we have the flexibility of adding new types without much code duplication.